projects/core/src/cart/store/selectors/multi-cart.selector.spec.ts

```diff
@@ -244,6 +270,22 @@ describe('Multi Cart selectors', () => {
     });
   });
 
+  describe('getCartsSelectorFactory', () => {
+    it('should return list of cart for entry', () => {
+      let result;
+      store
+        .pipe(select(MultiCartSelectors.getCartsSelectorFactory))
+        .subscribe((value) => (result = value));
+
+      const isEmpty = result === undefined || result.length === 0;
```

Comment:
Test should be clear (no conditions and this one can be considered as one). It is either undefined or empty array